tests/test_digital.py

```diff
@@ -5,13 +5,13 @@
 from qermod import (
     AnalogDepolarizing,
     Bitflip,
-    NoiseCategory,
+    Noise,
     PrimitiveNoise,
     deserialize,
     serialize,
 )
 
-list_noises = [noise for noise in NoiseCategory.DIGITAL]
+list_noises = [noise for noise in Noise.DIGITAL]
```

Comment:
```suggestion
digital_noises = Noise.DIGITAL.list()
```
Would work, _I think_.